apps/server/src/infra/etherpadClient/loggable/etherpad-cookies-configuration-missing.loggable.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { EtherpadConfigurationMissingLoggable } from './etherpad-configuration-missing.loggable';
+import {
+	EtherpadCookiesConfigurationMissingLoggable,
+	MissingCookie,
+} from './etherpad-cookies-configuration-missing.loggable';
+
+describe(EtherpadCookiesConfigurationMissingLoggable.name, () => {
+	describe('getLogMessage', () => {
+		describe('when cookie expiration missing', () => {
+			it('should return a log message', () => {
+				const cookieValue = 888;
+				const loggable: EtherpadConfigurationMissingLoggable = new EtherpadCookiesConfigurationMissingLoggable(
+					cookieValue,
+					MissingCookie.cookieExpiration
+				);
```

Comment:
Please move it to the `setup()` function.

Reply:
done